doc/cephadm/certmgr.rst

```diff
@@ -207,14 +215,12 @@ To associate a certificate with a private key:
 
 .. prompt:: bash #
 
-   ceph orch certmgr cert-key set <entity> [--cert <value>] [--key <value>] [--service_name <value>] [--hostname <value>] [-i <cert-key-path>] [--force]
+   ceph orch certmgr cert-key set <service-type> [--cert <value>] [--key <value>] [--service_name <value>] [--hostname <value>] [-i <cert-key-path>] [--force]
 
-Use this command to upload or replace an existing certificate/key pair
-for a certain service. Replace `<entity-name>` with the actual
-certificate name from `ceph orch certmgr entity ls`. The -i option
-can be used to specify a file containing a combined certificate and
-key in PEM format. This file should include both the certificate and
-private key concatenated together.
+Use this command to upload or replace an existing certificate/key pair for a certain service.
+Replace `<service-type>` with the actual certificate name from `ceph orch certmgr bindings ls`.
```

Comment:
Same as above.

```suggestion
Replace ``<service-type>`` with the actual certificate name from ``ceph orch certmgr bindings ls``.
```

Two more occurrences of single backticks used for this purpose in the following sections outside of this PR/in the old version of the file.